components/file-upload/src/index.js

```diff
@@ -93,10 +93,13 @@ FileUpload.propTypes = {
     active: PropTypes.bool,
     dirty: PropTypes.bool,
     dirtySinceLastSubmit: PropTypes.bool,
+    // eslint-disable-next-line react/forbid-prop-types
```

Comment:
these should be PropTypes.node I think - please could you open an issue to review these?

Reply:
added #794